DER/DerivativesContracts/ExoticOptions.rdf

```diff
@@ -150,6 +150,23 @@
 		<owl:disjointWith rdf:resource="&fibo-der-der-exo;LinearPayoutCommitment"/>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-der-der-exo;CapDetermination">
+		<rdfs:subClassOf rdf:resource="&fibo-der-der-exo;CapOrFloorDetermination"/>
+		<rdfs:label xml:lang="en">cap determination</rdfs:label>
+		<skos:definition xml:lang="en">The Strike rate is to be subtracted from the Observable rate on the reset date, to determine the settlement amount. This is incorrect. Cap applies to borrower. the Cap is the max amount I will pay to borrow. I take a capped rate. I am the buyer of the option. The market rate this time is higher than the capped rate. You as the seller of the option will pay me thedifference bwteen the capped rate and the market reate. If the market rate is lower than the capped rate no one pays anything because I will not exercise my right to that capped rate. So no money changes hands at settlement. Then: capped Loan: What I would be paying would be interest at the market rate if the market rate is lower, but if the market rate is higher then I would pay interest at the capped rate. This is the difference between a cap as a loan or as a cash settled thing. These represent different kinds of contract: Non loan one: Cap talks of settlement being difference between the market rate and the strike rate. This is cash settled. Name: &quot;Cap&quot; Loan one: Cap talks about a maximum interest rate payable by the holder, so the holder pays an amount each time, up to a maximum which is the Cap rate. Name: &quot;Capped Loan&quot; What manner of thin is this? A loan. Its ancestors are not OTC derivatives contracts thet are Loan.</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-der-der-exo;CapLoan">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-fi;DebtInstrument"/>
+		<rdfs:label xml:lang="en">cap loan</rdfs:label>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-der-der-exo;CapOrFloorDetermination">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-gao-obj;Strategy"/>
+		<rdfs:label xml:lang="en">cap or floor determination</rdfs:label>
+		<skos:definition xml:lang="en">Whether a given thing is treated as a Cap (something goes no higher than this amount) or as a Floor (something goes no lower than this).</skos:definition>
+	</owl:Class>
+	
 	<owl:Class rdf:about="&fibo-der-der-exo;CashOrNothingPayoutCommitment">
```

Comment:
This class is not needed: we don't model other required payments as "commitments"

Reply:
@rivettp See my comment above about not making changes to the content of the IROptions ontology, simply moving it straight across, for this particular pull request.